jobs/start-grunnlagsversjonering/README.md

```diff
@@ -0,0 +1,17 @@
+# start-grunnlagsversjonering
+___
+[Naisjob](https://doc.nais.io/naisjob/) for å poste en melding til kafka. 
+
+Benyttes for å starte versjonering av grunnlag.
+
+
+For å trigge denne jobben gjøres følgende:
+1. Finn docker image versjon som skal benyttes av denne jobben.
+   1. Gå til siste bygg i [GitHub actions](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pkgs/container/pensjon-etterlatte-saksbehandling%2Fstart-grunnlagsversjonering) og sjekk loggene for image versjon.
```

Comment:
Jeg trodde disse kunne deployes som vanlige apper direkte per miljø i stedet for dette?
Hvor brukes image versjonen? Skjønner ikke helt beskrivelsen her. Kan ikke se at den blir brukt.

Reply:
Bare copypaste fra `start-regulering` 🤷‍♂️  La null arbeid i readme siden dette skal slettes så fort jobben er kjørt OK.

Reply:
Aha kunne like gjerne slettet den da